lib/core/spec/dummy/config/locales/en.yml

```diff
@@ -31,3 +31,10 @@
 
 en:
   hello: "Hello world"
+  activemodel:
```

Comment:
Yes, it's needed cause we include ActiveModel::Validations. RSpec suggested to put the missing translation in this dummy part. Since we already have it `ros/lib/core/config/locales/en.yml`, I'll just move them there.